resources/sqlite/src/conn_opts.rs

```diff
@@ -0,0 +1,214 @@
+    pub fn read_only(mut self, read_only: bool) -> Self {
+        self.read_only = read_only;
+        self
+    }
+
+    /// Different to the original implementation, Shuttle defaults to `true` here.
+    /// See [sqlx docs](https://docs.rs/sqlx/latest/sqlx/sqlite/struct.SqliteConnectOptions.html#method.create_if_missing).
+    pub fn create_if_missing(mut self, create: bool) -> Self {
+        self.create_if_missing = create;
+        self
+    }
+
+    /// See [sqlx docs](https://docs.rs/sqlx/latest/sqlx/sqlite/struct.SqliteConnectOptions.html#method.statement_cache_capacity).
+    pub fn statement_cache_capacity(mut self, capacity: usize) -> Self {
+        self.statement_cache_capacity = capacity;
+        self
+    }
+
+    /// See [sqlx docs](https://docs.rs/sqlx/latest/sqlx/sqlite/struct.SqliteConnectOptions.html#method.busy_timeout).
+    pub fn busy_timeout(mut self, timeout: Duration) -> Self {
+        self.busy_timeout = timeout;
+        self
+    }
+
+    /// See [sqlx docs](https://docs.rs/sqlx/latest/sqlx/sqlite/struct.SqliteConnectOptions.html#method.immutable).
+    pub fn immutable(mut self, immutable: bool) -> Self {
+        self.immutable = immutable;
+        self
+    }
+
+    /// See [sqlx docs](https://docs.rs/sqlx/latest/sqlx/sqlite/struct.SqliteConnectOptions.html#method.serialized).
+    pub fn serialized(mut self, serialized: bool) -> Self {
+        self.serialized = serialized;
+        self
+    }
+
+    /// See [sqlx docs](https://docs.rs/sqlx/latest/sqlx/sqlite/struct.SqliteConnectOptions.html#method.command_buffer_size).
+    pub fn command_buffer_size(mut self, size: usize) -> Self {
+        self.command_channel_size = size;
+        self
+    }
+
+    /// See [sqlx docs](https://docs.rs/sqlx/latest/sqlx/sqlite/struct.SqliteConnectOptions.html#method.row_buffer_size).
+    pub fn row_buffer_size(mut self, size: usize) -> Self {
+        self.row_channel_size = size;
+        self
+    }
+
+    /// See [sqlx docs](https://docs.rs/sqlx/latest/sqlx/sqlite/struct.SqliteConnectOptions.html#method.vfs).
+    pub fn vfs(mut self, vfs_name: impl Into<Cow<'static, str>>) -> Self {
+        self.vfs = Some(vfs_name.into());
+        self
+    }
+}
+
+impl TryFrom<&ShuttleSqliteConnOpts> for SqliteConnectOptions {
+    type Error = shuttle_service::Error;
+
+    fn try_from(opts: &ShuttleSqliteConnOpts) -> Result<Self, Self::Error> {
+        let ShuttleSqliteConnOpts {
+            storage_path,
+            filename,
+            journal_mode,
+            synchronous,
+            in_memory,
+            read_only,
+            create_if_missing,
+            statement_cache_capacity,
+            busy_timeout,
+            immutable,
+            vfs,
+            serialized,
+            command_channel_size,
+            row_channel_size,
+        } = opts;
+
+        let db_path = storage_path.join(&filename);
```

Comment:
Only needed if in_memory is false. I'm also wondering: which folder is storage_path pointing to currently (locally vs when deployed)?